qemu/tests/cfg/block_disk_not_default_options.cfg

```diff
@@ -0,0 +1,20 @@
+- block_disk_not_default_options:
+    only virtio_blk, virtio_scsi
+    type = block_disk_not_default_options
+    only x86_64
+    start_vm = yes
+    images += " stg0"
+    boot_drive_stg0 = yes
+    image_name_stg0 = images/storage0
+    image_size_stg0 = 40G
+    remove_image_stg0 = yes
+    force_create_image_stg0 = yes
+    dd_cmd = "dd if=/dev/urandom of=test.txt bs=1M count=10"
+    variants:
+        - with_packed_and_ppv:
+            qtree_check = '{"packed" : ["virtio-blk-device", "virtio-scsi-device"], "page-per-vq" : ["virtio-blk-pci", "virtio-scsi-pci"]}'
```

Comment:
1. it can be evolved to support different image ,like as
qtree_check_stg0.....
2. where is the value definition ?example, I want to check the key1 value is "ABC"?